networkx/algorithms/centrality/tests/test_voterank.py

```diff
@@ -59,3 +102,12 @@ def test_voterank_centrality_5(self):
         )
         exact = [2, 0, 5, 4]
         assert exact == nx.voterank(G)
+
+    # MultiDiGraph unit test
+    def test_vote_rank_centrality_5(self):
```

Comment:
What is this testing? Put that into the test name rather than a number. Then when a test says that `test_vote_rank_centrality_for_MultiDiGraph` failed, we easily know what the test is about. (and you can remove the comment).  Tests with numbers in the name are not as helpful.